mysite/views.py

```diff
@@ -1,3 +1,175 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect, get_object_or_404
+from django.contrib.auth.models import User
+from django.contrib.auth import authenticate, login, get_user_model
+from django.http import HttpResponseRedirect
+from django import forms
+from django.urls import reverse
+from django.views.generic import UpdateView
+from django.contrib import messages
+from django.contrib.auth import update_session_auth_hash
+from django.contrib.auth.forms import PasswordChangeForm
+from mysite.forms import UserRegistrationForm, UserProfileForm, ProjectForm, ProjectBidForm, AssignProjectForm
+from mysite.models import Project, ProjectBid
 
-# Create your views here.
+User = get_user_model()
+
+
+def home(request):
+    return render(request, 'mysite/home.html')
```

Comment:
Please put some description for each method at first line in method. e;g
def home(request):
    """ render home page to show project list """
    // All other code goes here

Do this change for all method either for views/helper methods.

Reply:
Feedback Incorporated.